### PR TITLE
Feature/error handling

### PR DESCRIPTION
**Pull request by SamSalvatico**
### Description

Added an `error-handling` package to normalize handling across packages

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [x] **New feature**
- [ ] **Dev change**

### Checklist:

- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: apps/payments-api/app.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { dirname, join } from "path";
 import healthCheck from "./routes/healthcheck";
 import sensible from "@fastify/sensible";
 import schemaValidators from "./routes/schemas/validations";
+import { initializeErrorHandling } from "error-handling";
```
Comment (GiuliaTeggi) [inline apps/payments-api/app.ts:17]: Super nitpick, but I would call this error-handlER (to follow the same pattern of logging-wrappER)

**Review by GiuliaTeggi — APPROVED** [review]
LGTM 😄  Great job on these utils!!

**Review by Thurfjell — COMMENTED** [review]
Q out of ignorance and not understanding. 
What, how and where are we logging in ie. prod?

File: apps/payments-api/app.ts (hunk 1 above)
Comment (SamSalvatico) [reply apps/payments-api/app.ts]: You're totally right, thank you Giulia!

**Comment by SamSalvatico** [comment]
> Q out of ignorance and not understanding. What, how and where are we logging in ie. prod?

We will log same entries in whole environments. The only thing that will change is the minimum logged level.
e.g. we can decide to not log `DEBUG` level on prod

https://github.com/ogcio/life-events/blob/feature/logging-wrapper-tests/packages/logging-wrapper/README.md#default-records

**Review by Thurfjell — APPROVED** [review]

**Pull request merged**